test/relations.test.js

```diff
@@ -2152,11 +2250,21 @@ describe('relations', function() {
       Picture.find({include: 'imageable'}, function(err, pics) {
         should.not.exist(err);
         pics.should.have.length(2);
-        pics[0].name.should.equal('Author Pic');
-        pics[0].imageable().name.should.equal('Author 1');
-        pics[1].name.should.equal('Reader Pic');
-        pics[1].imageable().name.should.equal('Reader 1');
-        done();
+        if (isCassandraConnector) {
+          // Sorting is not set in define for Cassandra
+          var names = ['Author Pic', 'Reader Pic'];
+          var imageables = ['Author 1', 'Reader 1'];
+          names.should.containEql(pics[0].name);
+          names.should.containEql(pics[1].name);
+          imageables.should.containEql(pics[0].imageable().name);
+          imageables.should.containEql(pics[1].imageable().name);
+        } else {
+          pics[0].name.should.equal('Author Pic');
+          pics[0].imageable().name.should.equal('Author 1');
+          pics[1].name.should.equal('Reader Pic');
+          pics[1].imageable().name.should.equal('Reader 1');
+        }
+        done(err);
```

Comment:
Just `done()`. Errors have been already handled at the top of this callback function.